Add link to package hub disclaimer page (#5122)
fishtown-analytics/docs.getdbt.com: website/docs/docs/build/packages.md

You can specify a package using one of the following methods, depending on where your package is stored.

### Hub packages (recommended)

> 

Reviewer: remoe the callout as i think it should be its own sentence here introducing this section. plus it was rendering weird bc of the spacing. let me know if yo have strong opinions though!
![Screenshot 2024-03-21 at 10 23 16](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/c28c2d6d-75f9-4615-a57e-75db01e3687c)

Reviewer: update now looks like this:
![Screenshot 2024-03-21 at 11 22 13](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/c0a5bed6-5e61-4113-97c3-474f294faf7c)

> dbt Labs hosts the [Package hub](https://hub.getdbt.com) as a courtesy to the dbt Community, but does not certify or confirm the integrity, operability, effectiveness, or security of any Packages. Please read the [dbt Labs Package Disclaimer](https://hub.getdbt.com/disclaimer/) before installing Hub packages.

Reviewer: ```suggestion
dbt Labs hosts the [Package hub](https://hub.getdbt.com), registry for dbt packages, as a courtesy to the dbt Community. The Package hub does not certify or confirm the integrity, operability, effectiveness, or security of any Packages. Please read the [dbt Labs Package Disclaimer](https://hub.getdbt.com/disclaimer/) before installing Hub packages.
```

> dbt Labs hosts the [Package hub](https://hub.getdbt.com), registry for dbt packages, as a courtesy to the dbt Community. dbt Labs does not certify or confirm the integrity, operability, effectiveness, or security of any Packages. Please read the [dbt Labs Package Disclaimer](https://hub.getdbt.com/disclaimer/) before installing Hub packages.

Reviewer: ```suggestion
dbt Labs hosts the [Package hub](https://hub.getdbt.com), registry for dbt packages, as a courtesy to the dbt Community, but does not certify or confirm the integrity, operability, effectiveness, or security of any Packages. Please read the [dbt Labs Package Disclaimer](https://hub.getdbt.com/disclaimer/) before installing Hub packages.
```

> dbt Labs hosts the [Package hub](https://hub.getdbt.com), registry for dbt packages, as a courtesy to the dbt Community. The Package hub does not certify or confirm the integrity, operability, effectiveness, or security of any Packages. Please read the [dbt Labs Package Disclaimer](https://hub.getdbt.com/disclaimer/) before installing Hub packages.

Reviewer: ```suggestion
dbt Labs hosts the [Package hub](https://hub.getdbt.com), registry for dbt packages, as a courtesy to the dbt Community. dbt Labs does not certify or confirm the integrity, operability, effectiveness, or security of any Packages. Please read the [dbt Labs Package Disclaimer](https://hub.getdbt.com/disclaimer/) before installing Hub packages.
```

> You can install available hub packages listed in the Package hub in the following way:

Reviewer: ```suggestion
You can install available hub packages in the following way:
```
